### ISSUE TITLE
Escape button shouldn't be a list item

### ISSUE THREAD

**Issue by BryceStevenWilley** [SuffolkLITLab/docassemble-ALToolbox#196]
Currently, the escape button shows up in the nav bar as a list item (`<li>`), not in any surrounding HTML list. This is generally an accessibility issue, as screen readers don't know how to handle list items just hanging around like that, and also has the default bullet, visible with different nav bar background colors (seen below). 

![A screenshot of the docassemble nav bar. The background is dark blue, there is a court forms online logo, and very close to the logo is a black bullet point, and a red escape button](https://github.com/SuffolkLITLab/docassemble-ALToolbox/assets/6252212/8618819e-49c3-4764-909a-cbf9cc9e2ae8)

Easy fix should be simple; just make a `div` instead of an `li`. There might be better HTML for that, but not sure of it off the top of my head.
